.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": "/Users/lf/.pyenv/versions/mxnet/bin/python"
```

Comment:
put .vscode extension file in .gitignore

Reply:
It is still showing in the change list .. So, you need to also remove it from the git cache since you have already pushed it before